contracts/contracts/GrantRound.sol

```diff
@@ -24,9 +24,12 @@ contract GrantRound {
   /// @notice GrantsRegistry
   GrantRegistry public immutable registry;
 
-  /// @notice ERC20 token that accepts pool donations
+  /// @notice ERC20 token that accepts donations
```

Comment:
I think this comment for the `donationToken` could be a bit more clear. Maybe

This comment a bit unclear
```suggestion
  /// @notice Token used for all contributions. Contributions in a different token are swapped to this token
```